soccer/gameplay/tactics/pass_to_point_heuristic.py

```diff
@@ -0,0 +1,7 @@
+import constants
+import robocup
+import time
+
+
+def pass_to_point_heuristic(x: int, y: int) -> robocup.Point:
```

Comment:
What's the thought behind doing (x, y) instead of point?

Reply:
It's just a way to test stuff and as a template for new members for heuristics, like since we aren't making any changes to coordinated pass, we would just pass a point straight into it like we already do if we wanted to pass to a point.

Reply:
So...... what's the point of passing in the `x` and `y` coordinates on the field?

Reply:
What is this supposed to represent?

Reply:
Its where you want to pass to, like this isn't really meant to be a real heuristic we use just like a thing for new members to look at so they what one should look like. Like if I just passed in a point and immediately returned it wouldn't be helpful for new members to look at.

Reply:
So..... how are you supposed to know where you want to pass to if the heurustic is telling you where to pass to?

Reply:
The heuristic isn't. The heuristic in this case is just a function that takes in an x and y coordinate that you feed it when writing the play and returns a point. In an actual useful heuristic that a freshman would write they would pass in like other stuff, like maybe acceptable risk of pass or something,  and create a function that returns a point of where to pass to, which is then passed into coordinated pass.